ISISScientificComputing/autoreduce
# 37 Fix reduction run summary routing to use instrument name

**Description of changes**


- Adds the instrument name to URL when viewing the reduction run summary so if two instruments ever have the same run number then they won't clash
- Also adds a URL route so the default run version is 0. So you can, for example, go to `/runs/ORISIS/130777/` and it will take you to the first run version of 130777.

**How to test**


- Check the web app works as expected with no 500/404 errors on any links.
- Check the new URL routing actually works
    - Go to `/runs/<runnumber>/<runversion>` - this should take you to a 404.
    - Go to `/runs/<instrument>/<runnumber>/<runversion>` - this should take you to the correct run summary.
    - Go to `/runs/<instrument>/<runnumber>` - this should take you to the summary for the first run version of this run number on this instrument.


Fixes #37 

---


**To test**
* Log into the devolpement nodes
* `git checkout this-pull-request-branch`
* restart services on each node (information on how to do this can be found in the development documentation)
* Submit some runs with the manualsubmission.py script

Once you are happy, merge this pull request into develop, and update the development nodes to the devlop branch
```
> git checkout origin/devlop
> git fetch -p
> git pull origin/develop
```
